### PR TITLE
Soukainahn submission 

### PR DESCRIPTION
**Pull request by soukainahn**

### CODE REVIEW

File: submissions/soukainahn/mycode.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
+#include <iostream>
```
Comment (aboullaite) [inline submissions/soukainahn/mycode.cpp:1]: Please keep only file submission and remove everything else :) thank you

**Review by aboullaite — CHANGES_REQUESTED** [review]

File: submissions/soukainahn/mycode.cpp (hunk 1 above)
Comment (soukainahn) [reply submissions/soukainahn/mycode.cpp:1]: could you please close the other pull request that i opened ,im not  very familiar with this paltform thank you so much

Comment (soukainahn) [reply submissions/soukainahn/mycode.cpp:1]: do you mean only keeping the source code file

Comment (aboullaite) [reply submissions/soukainahn/mycode.cpp:1]: exactly yes :)

**Pull request closed without merging**